sphinx/source/models.rst

```diff
@@ -169,6 +169,7 @@ Inducing Point Allocators
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .. automodule:: botorch.models.utils.inducing_point_allocators
     :members:
+    :private-members:
```

Comment:
Needed so that `_pivoted_cholesky_init` and related discussion of [burt2020svgp] and [chen2018dpp] show up